doc/content/examples/example6a.md

```diff
@@ -0,0 +1,65 @@
+# Example 6a: Displacement Controlled Contact Problem with elastic soil
+
+## Model Description
+
+This example demonstrates the use of the contact block to simulate a frictional contact between a solid block and an elastic soil in MASTODON. A solid cubical block of dimensions 6" each side  is placed in contact with an cubical soil block of dimensions 48" each side. The remaining three sides of the soil are fixed and the solid block is displaced by a prescribed distance in the direction parallel to the contact surface. The normal forces and the frictional forces obtained from MASTODON is compared with the analytical results.
+
+
+## Modeling in MASTODON
+
+!listing examples/ex06a/psf_grav.i
+
+The mesh consisting of the solid block and the soil is generated by importing an existing mesh file [fig:ex06input].
+The details about the type of mesh file supported can be found at [GettingStarted](manuals/user/index.md). The displacements are defined in the variables section and the velocity, acceleration, normal forces and the tangential (frictional) forces are defined as the auxiliary variables.
+
+!media media/examples/ex06/ex06_input.png
+       style=width:50%;margin-left:150px;float:center;
+       id=fig:ex06input
+       caption=Input model in MASTODON.
+
+
+[TensorMechanics](syntax/index.md) and [InertialForce](syntax/index.md) kernels are used to model the dynamics of the 3D solid elements. [Gravity](syntax/index.md) kernels is used to apply the gravitational force. The [Newmark time integration](manuals/theory/index.md) parameters used in this problem correspond to the Newmark's average acceleration method, i.e. `beta = 0.25` and `gamma = 0.5` and damping is ignored.
+
+The [ComputeIsotropicElasticityTensorBeam](syntax/index.md) block is used to create the elasticity tensor of the solid block and the elastic soil, using the parameters, Young's Modulus and Poisson's ratio. The stresses and strain is calculated using [ComputeFiniteStrainElasticStress](syntax/index.md) and [ComputeFiniteStrain](syntax/index.md). The densities are assigned to the solid block and the soil using [GenericConstantMaterial](syntax/index.md).
```

Comment:
...block is used to create the elasticity tensor of the solid block and the elastic soil using Young's Modulus and Poisson's ratio. The stresses and strains are calculated using